tests/moe_old.py

```diff
@@ -0,0 +1,216 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# --------------------------------------------------------------------------
```

Comment:
this file seems to be an older version of moe and possibly does not have any tests. May we remove it from here?

Reply:
Thanks for careful review. Removed now

Reply:
I added it back. Current test actually depend on this file.